showcases/screen-reader/default.ts

```diff
@@ -35,7 +35,8 @@ const standardPhrases = [
 const flakyExpressions: Record<string, string> = {
 	'pop-up': 'pop up',
 	'checked. checked': 'checked',
-	'selected. selected': 'selected'
+	'selected. selected': 'selected',
+	'expanded. expanded': 'selected'
```

Comment:
```suggestion
	'expanded. expanded': 'expanded'
```

this needs to be _expanded_, doesn't it ?

Reply:
I'm not sure :(

Reply:
🤫